pkg/qdr/qdr.go

```diff
@@ -448,6 +448,7 @@ type HttpEndpoint struct {
 	EventChannel    bool   `json:"eventChannel,omitempty"`
 	HostOverride    string `json:"hostOverride,omitempty"`
 	SslProfile      string `json:"sslProfile,omitempty"`
+	VerifyHostname  *bool  `json:"verifyHostname,omitempty"`
```

Comment:
@fgiorgetti I have added the `verifyHostname` field as suggested. Thanks!

cc:/@grs @ajssmith